src/MockPlayerBridge.ts

```diff
@@ -41,6 +76,85 @@ export class MockPlayerBridge extends AbstractPlayerBridge {
     return 'dummy-stream-id';
   }
 
+  protected pause() {
+    this.playbackState = 'PAUSED';
+    document.dispatchEvent(new Event('livery-playback-change'));
+  }
+
+  protected play() {
+    this.playbackState = 'PLAYING';
+    document.dispatchEvent(new Event('livery-playback-change'));
+  }
+
+  protected reload() {
+    this.playbackState = 'PLAYING';
+    document.dispatchEvent(new Event('livery-playback-change'));
+  }
+
+  protected seek(position: number) {
+    this.playbackState = 'SEEKING';
+    document.dispatchEvent(new Event('livery-playback-change'));
+    return position;
+  }
+
+  protected selectQuality(index: number) {
+    return index;
+  }
+
+  protected setControlsDisabled(disabled: boolean) {
+    return disabled;
+  }
+
+  protected setDisplay(display: DisplayMode) {
+    this.display = display;
+    document.dispatchEvent(new Event('livery-display-change'));
+    return display;
+  }
+
+  protected setMuted(muted: boolean) {
+    this.muted = muted;
+    document.dispatchEvent(new Event('livery-muted-change'));
+    return muted;
+  }
+
+  protected submitUserFeedback(payload: UserFeedbackPayload) {
+    return payload;
+  }
+
+  protected subscribeConfig(listener: (value: Config) => void) {
+    const config: Config = {
+      controls: {
+        cast: false,
+        contact: false,
+        error: false,
+        fullscreen: false,
+        mute: false,
+        pip: false,
+        play: false,
+        quality: false,
+        scrubber: false,
+      },
+      customerId: 'id',
+      streamPhase: 'LIVE',
+      streamPhases: ['LIVE', 'POST', 'PRE'],
+      tenantId: 'id',
+    };
+    listener(config);
```

Comment:
This listener call with the same value as returned below is useless. Instead what you could do here like I've done elsewhere is to arrange for a series of mock config values to be called back with through the listener over time.

Or, again as a nice to have as mentioned above, enable controlling the mock config from within the element, e.g: add a Mock Config panel.

The same goes for the mocked subscribe methods below; they should either just result in a prepared series of states to be sent to the listeners or they should be bound to state that can be modified from this element.

Reply:
This listener call should still be removed.